examples/example-gateway/idl/clients/baz/baz.thrift

```diff
@@ -14,6 +14,16 @@ struct BazRequest {
     3: required i32 i3
 }
 
+struct transHeaderType {
```

Comment:
Shouldn't we cover all [thrift base types](https://thrift.apache.org/docs/types) ?

Reply:
I'll raise panic to cover all the types, not all types apply to headers